### PR TITLE
fix: twitter card

### PR DESCRIPTION
**Pull request by Jefiozie**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature (please, look at the "Scope of the project" section in the README.md file)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/xlayers/src/index.html [hunk 1]
```diff
@@ -38,7 +38,7 @@
 
   <!-- twitter meta -->
   <meta name="twitter:title" content="xLayers (by Wassim Chegham)">
```
Comment (manekinekko) [inline packages/xlayers/src/index.html]:
```suggestion
  <meta name="twitter:title" content="xLayers: Design – Code – Repeat">
```

**Comment by Jefiozie** [comment]
Will fix the card problem : 
image


image

File: packages/xlayers/src/index.html (hunk 1 above)
Comment (Jefiozie) [reply packages/xlayers/src/index.html]: will do 👍

**Comment by Jefiozie** [comment]
Looks like something is changed and now `netlify` timedout: 

```
'building site': Command did not finish within the time limit
```

**Pull request merged**